src/components/Footer/Footer.test.jsx

```diff
@@ -0,0 +1,28 @@
+import Footer from "./Footer.jsx";
+import { describe, it, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+
+describe("Footer component", () => {
+  it("renders correctly at the bottom of the page", () => {
+    render(<Footer />);
+
+    const footerElement = screen.getByTestId("footer");
+    expect(footerElement).toBeInTheDocument();
+
+    // codes from line 14 to 17 has a bug
+
+    // const footerStyles = window.getComputedStyle(footerElement);
+    // console.log("here", footerStyles.position);
+    // expect(footerStyles.position).toBe("fixed");
+    // expect(footerStyles.bottom).toBe("0");
```

Comment:
I rember we discussed this bit of code in the class. I should have flagged that it's best to clean up commented out code when you can if it doesn't serve a purpose. Wouldn't take testing too seriously at this stage either they're just trying to get you into a habit, so there's no point spending ages on trying to write a perfect test, just write something :)